lab_2/main.py

```diff
@@ -24,7 +38,17 @@ def get_language_profiles(texts_corpus: list, language_labels: list) -> dict or
     :param language_labels: a list of given language labels
     :return: a dictionary of dictionaries - language profiles
     """
-    pass
+    if (not isinstance(texts_corpus, list)) or (not isinstance(language_labels, list)):
+        return None
+    language_profiles = {}
+    dictionary_lst = []
+    for text in texts_corpus:
+        if not isinstance(text, list):
+            return None
+        dictionary_lst.append(get_freq_dict(text))
+    for label in enumerate(language_labels):
```

Comment:
`enumerate` is usually used as follows: `for index, element in enumerate(some_iterable)` so that you do not have to use indexes inside the loop body.